src/components/10-atoms/radio/demo.js

```diff
@@ -81,36 +81,45 @@ storiesOf('Atoms/Radio/Demos', module)
 </svg>`;
 
     const template = html`
-    <axa-fieldset horizontal>
-    <axa-radio
-        button
-        name="insurance"
-        label="Option 1"
-        .icon="${icon}"
-    ></axa-radio>
-    <axa-radio
-        button
-        name="insurance"
-        label="Option 2"
-        .icon="${icon}"
-    ></axa-radio>
-    <axa-radio
-        button
-        name="insurance"
-        label="Option 3"
-        checked
-        .icon="${icon}"
-    ></axa-radio>
-    <axa-radio
-        button
-        name="insurance"
-        label="Option 4"
-        disabled
-        .icon="${icon}"
-    ></axa-radio>
-    </axa-fieldset>
-`;
+      <axa-fieldset horizontal>
+        <axa-radio
+          button
+          name="insurance"
+          label="Option 1"
+          .icon="${icon}"
+        ></axa-radio>
+        <axa-radio
+          button
+          name="insurance"
+          label="Option 2"
+          .icon="${icon}"
+        ></axa-radio>
+        <axa-radio
+          button
+          name="insurance"
+          label="Option 3"
+          checked
+          .icon="${icon}"
+        ></axa-radio>
+        <axa-radio
+          button
+          name="insurance"
+          label="Option 4"
+          disabled
+          .icon="${icon}"
+        ></axa-radio>
+      </axa-fieldset>
+    `;
 
     render(template, root);
     return root;
-  });
+  })
+  .add(
+    'Radio - no label',
+    () => `
+      <axa-fieldset>
```

Comment:
https://axa-ch.github.io/plib-feature/bugfix/double-label-radio/?path=/story/atoms-radio-demos--radio-no-label

This example does not work.

Reply:
True, I will check it out.

Reply:
I deleted the demo.